website/markdown/docs/commands/dependencies.mdx

```diff
@@ -0,0 +1,211 @@
+---
+name: Manage third-party dependencies
+excerpt: "Learn how to use Tuist's dependencies command to manage third-party dependencies."
+---
+
+import PropertiesTable from '../components/properties-table'
+import SettingsDictionaryTable from '../components/settingsdictionary-table'
+import Message from '../components/message'
+import EnumTable from '../components/enum'
+import {
+  List,
+  ListItem,
+  ListIcon,
+  ListContent,
+  ListHeader,
+  ListDescription,
+  ListList,
+} from '../components/list'
+
+# Dependencies (Chimera)
+
+## How to get started
```

Comment:
I think it'd be great to do a step-by-step  example how you could add eg Carthage dependency, install it and integrate it with the project. Users usually want to get started quickly before reading long documentation and they are used to learn by examples.

Reply:
Love the idea. As a user, if you are guiding through a process the experience improves significantly.